include/engine/api/route_api.hpp

```diff
@@ -235,6 +235,14 @@ class RouteAPI : public BaseAPI
                 auto &leg_geometry = leg_geometries[idx];
                 util::json::Object annotation;
 
+                if (parameters.annotations_type & RouteParameters::AnnotationsType::Speed)
+                {
+                    annotation.values["speed"] = GetAnnotations(
+                        leg_geometry, [](const guidance::LegGeometry::Annotation &anno) {
+                            return anno.distance / anno.duration;
```

Comment:
this will give misleading speeding precision, because duration is 0.1 accurate. probably
```
std::round(10. * anno.distance / anno.duration) / 10.
```
is good enough.